tests/modules/create_test_yml.py

```diff
@@ -1,13 +1,14 @@
 import os
-import tempfile
 import pytest
 
 import nf_core.modules
 
+from ..utils import with_temporary_folder
 
-def test_modules_custom_yml_dumper(self):
+
+@with_temporary_folder
+def test_modules_custom_yml_dumper(self, out_dir):
```

Comment:
This is the decorator which implements the context manager. The temporary folder name is added as parameter. I try to use the same names when possible in order to minimize changes in code